### PR TITLE
Fixed width for suggested replies text box

### LINKED ISSUE #59: SMS edit boxes are not fixed width
Screenshot 2020-12-07 at 20 53 42

### PR DESCRIPTION
**Pull request by ktkization**
Fixes #59

### CODE REVIEW

File: webapp/web/configuration.css [hunk 1]
```diff
@@ -206,8 +206,9 @@
   overflow-y: scroll;
   border: 1px solid #ddd;
   border-radius: .5rem;
-  padding: 2px;
-  min-height: 70px;
+  padding: 3px;
+  width: 95%;
+  height: 75px;
```
Comment (marianamarasoiu) [inline webapp/web/configuration.css]: I see this changed from `min-height` to `height`, but I don't think it's what we want? the previous behaviour in terms of height is what we want - start with a minimum, and expand based on contents, so can you please revert this? thanks!

Comment (ktkization) [reply webapp/web/configuration.css]: Reverted it back

Comment (marianamarasoiu) [reply webapp/web/configuration.css]: thanks! unfortunately it doesn't seem to behave as it did - if I write something, the suggested replies box I'm writing in is expanding correctly, but the translation is somehow shorter height, and expands slower. I think they should all be the same height if possible?

Comment (ktkization) [reply webapp/web/configuration.css]: @marianamarasoiu  I've been able to make them expand at the same rate

**Review by marianamarasoiu — APPROVED** [review]
LGTM, thanks!

**Pull request merged**